api/src/pcapi/core/offerers/synchronize_venues_banners_with_google_places.py

```diff
@@ -167,7 +167,8 @@ def synchronize_venues_banners_with_google_places(
                 if (datetime.datetime.utcnow() - venue.googlePlacesInfo.updateDate).days > 62:
                     continue
             else:
-                place_id = get_place_id(venue.name, venue.street, venue.city, venue.postalCode)
+                name = venue.publicName if venue.publicName else venue.name
+                place_id = get_place_id(name, venue.street, venue.city, venue.postalCode)
```

Comment:
On a la propriété `common_name` faite pour ça :slightly_smiling_face: 
```suggestion
                place_id = get_place_id(venue.common_name, venue.street, venue.city, venue.postalCode)
```
Ça peut valoir le coup d'utiliser `publicName` dans l'un des tests auto de cette fonctionnalité.

Reply:
on a pas de tests auto pour get_place_id puisque ça fait appel à google principalement.

Reply:
On ne vérifie pas les paramètres qu'on envoie à Google ??